packages/peregrine/lib/List/list.js

```diff
@@ -1,5 +1,13 @@
 import React, { useMemo, useCallback } from 'react';
-import PropTypes from 'prop-types';
+import {
```

Comment:
Not sure where to comment this, but does `items` need to exist or can it be folded into the list component?

Reply:
These changes are no longer part of this PR